docs/static/Devel/UD_Viz_Browser/SPARQL/Readme.md

```diff
@@ -11,23 +11,23 @@ The basic functionalities of the SPARQL module include:
 
 ## Usage
 
-For an example of how to the SPARQL Widget to a UD-Viz web application see the [SPARQLWidget example](https://github.com/VCityTeam/UD-Viz/blob/master/packages/browser/examples/SPARQLWidget.html)
+For an example of how to the SPARQL Widget to a UD-Viz web application see the [SPARQLWidget example](https://github.com/VCityTeam/UD-Viz/blob/master/examples/SPARQLWidget.html)
```

Comment:
The pointed markdown seems to live in this repo. Why is this not a relative link ?

Reply:
All .md in docs/static are going to generate an html page in the online documentation, if the link is relative it will be broken online. That's why I think we should prefer url. Note that all .md are now going to be consider by jsdoc we should then replace all links by URL ? What do you think ?

Reply:
What about the case when we add a link to a new document that only exists in a local commit or branch. Here a link towards the online jsdoc would be broken.

This might be an edge case, but I think we need to define how to resolve it.

Reply:
Right, the CI command will not pass since these links do not exist yet. To overcome this we could adjust FAILURE_THRESHOLD variable to temporary pass the CI command. But it's quite an hack since we should then reset everytime the FAILURE_THRESHOLD variable...

Reply:
Just a thought... what about before to launch remark validate links we replace prefix "https://github.com/VCityTeam/UD-Viz/blob/master/" by "http:localhost:8000/" and we launch a local server on the UD-Viz repo. Url should be fine then we undo the prefix modification ?

Still sound like a hack but it should works, what about that ?

Reply:
I have created an issue #562 to think about how to solve the online documention link problem.
Since this PR is solving all dead links in github/desktop environment we could close it (even if some url could be relative path, this problem belong to #562) @DiegoVinasco and @EricBoix do you agree ?

Reply:
Yeah this issue can easily fall outside the scope of the PR. Sounds good to me.